tests/refs/scope_draw_histogram_luma_only.ref

```diff
@@ -0,0 +1,5 @@
+549D75DD35D17DDC39917DD428817DD5 549D75DD35D17DDC39917DD428817DD5 549D75DD35D17DDC39917DD428817DD5 00000000000000000000000000000000
+553D55D1559D4DDC3D917DD428817DD5 553D55D1559D4DDC3D917DD428817DD5 553D55D1559D4DDC3D917DD428817DD5 00000000000000000000000000000000
+54B575D545D57DD439957DD028817DD5 54B575D545D57DD439957DD028817DD5 54B575D545D57DD439957DD028817DD5 00000000000000000000000000000000
+553D55F555914DDC3D916DD468807DD7 553D55F555914DDC3D916DD468807DD7 553D55F555914DDC3D916DD468807DD7 00000000000000000000000000000000
+55DD55D575916DDC39917DD428817DD5 55DD55D575916DDC39917DD428817DD5 55DD55D575916DDC39917DD428817DD5 00000000000000000000000000000000
```

Comment:
You need to remove `tests/refs/scope_render_histogram_luma_only.ref`

Reply:
Done.